src/scores/continuous/__init__.py

```diff
@@ -1,7 +1,10 @@
 """
 Import the functions from the implementations into the public API
 """
-from scores.continuous.flip_flop_impl import flip_flop_index
+from scores.continuous.flip_flop_impl import (
+    flip_flop_index,
+    flip_flop_index_proportion_exceeding,
```

Comment:
I think we also need to add it here https://github.com/nci/scores/blob/develop/docs/api.md